### PR TITLE
chairs don't immediately vacate upon election

### PR DESCRIPTION
**Pull request by npdoty**
no more than one election every year
possible resolution of issue #1

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]

**Review by eriktaubeneck — APPROVED** [review]
I support this change.

**Comment by AramZS** [comment]
This seems reasonable to me!

**Comment by AramZS** [comment]
Change to 3 months then merge.

File: charter.html [hunk 1]
```diff
@@ -311,10 +311,11 @@ <h3 id=chair-selection>Chair Selection</h3>
 
 <p>Chairs are added by election if the number of Chairs is reduced below two.
 Alternatively, if five Community Group Participants &mdash; no two from the same
-organization &mdash; call for an election, then all current chairs must vacate
-their position.
+organization &mdash; call for an election, and no election has taken place in
+the previous 12 months, then an election is held to select two Chairs in place
```
Comment (martinthomson) [inline charter.html:315]:
```suggestion
the previous three months, then an election is held to select two Chairs in place
```

**Pull request merged**